config/webpack.config.dev.js

```diff
@@ -0,0 +1,24 @@
+const merge = require('webpack-merge');
+const path = require('path');
+const Dotenv = require('dotenv-webpack');
+const commonConfig = require('./webpack.config.common');
+
+module.exports = merge(commonConfig, {
+  mode: 'development',
+  devtool: 'eval-source-map',
```

Comment:
lets use `inline-source-map`, which is like the dev version of prod relative `source-map` in the sense that it gives our original source code and not generated one

Reply:
I saw it in the Webpack doc that they both generate original source code which is essential for debugging our app, and they are both slow when they build for the first time.  The reason I opted in for the eval-source-map was that eval-source-map is a lot faster when rebuilding. Here is the doc I used https://webpack.js.org/configuration/devtool/#devtool